x/ibc/03-connection/keeper/verify.go

```diff
@@ -31,9 +31,16 @@ func (k Keeper) VerifyClientConsensusState(
 		return sdkerrors.Wrapf(clienttypes.ErrConsensusStateNotFound, "clientID: %s with height: %d", clientID, height)
 	}
 
-	return clientState.VerifyClientConsensusState(
+	clientState, err = clientState.VerifyClientConsensusState(
```

Comment:
```suggestion
	clientState, err := clientState.VerifyClientConsensusState(
```

Then  you can remove the var from the return.